src/platform/silabs/KeyValueStoreManagerImpl.cpp

```diff
@@ -299,6 +317,36 @@ void KeyValueStoreManagerImpl::KvsMapMigration(void)
     }
 }
 
+void KeyValueStoreManagerImpl::KvsKeyMapCleanup(void * argument)
+{
+    bool requireKvsKeyMapSave = false;
+
+    for (uint16_t key = 0; key < KeyValueStoreManagerImpl::kMaxEntries; key++)
```

Comment:
How long does this for loop takes if kvsKeyMap is full and kMaxEntries is 511 (our current absolute max)

Reply:
- Under 40 ms when KeyMap is fully used by shadow keys. 
- Under 1 ms after a single commissioning / unpair
- No shadow keys, approx 0